app/domain/accounts/router.py

```
@@ -20,3 +23,14 @@ async def create_account(
 ):
 
     return await service.create_update(payload, user)
+
+
+@router.get("", response_model=Page[AccountResponse])
+async def list_accounts(
+    financial_institution_id: uuid.UUID | None = None,
+    status: AccountStatus | None = None,
+    type: AccountType | None = None,
+    service: AccountService = Depends(get_account_service),
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)